src/index.js

```diff
@@ -6,18 +6,21 @@ import { makeServer } from "./server";
 import { ProductListProvider } from "./context/product-context";
 import { AuthProvider } from './context/auth-context'
 import { WishListProvider } from "./context/wishlist-context";
+import { CartProvider } from "./context/cart-context";
```

Comment:
Try using index files for importing components and context

Reply:
Sure will do that